var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -531,6 +535,8 @@ class Openmpi(AutotoolsPackage, CudaPackage):
     # Singularity release 3 works better
     depends_on("singularity@3:", when="+singularity")
     depends_on("lustre", when="+lustre")
+    depends_on("ucx +rocm", when="+rocm")
```

Comment:
we can't take this.  a number of important sites are using MI250/300 and we don't have UCX anywhere on those systems.